### PR TITLE
CS: Fix PHPDoc format

### PR DESCRIPTION
**Pull request by dominics**
- PHPDoc comments should begin with `/**` to distinguish them from regular block comments. Otherwise IDEs and static analysis tools will ignore them.
- `Array` is a type, not a class in PHP (should not ever be capitalized)

This is a BC coding standards path. It should have no functional changes.

### CODE REVIEW

**Comment by nietonfir** [comment]
This PR also greatly improves the readability of the documentation imho.

File: src/Raygun4php/RaygunClient.php [hunk 1]
```diff
@@ -429,16 +439,16 @@ function filterParamsFromMessage($message, $replace = '[filtered]') {
     }
 
     /**
-     * @param Array $params
-     * @return Raygun4php\RaygunClient
+     * @param array $params
+     * @return self
```
Comment (robbieaverill) [inline src/Raygun4php/RaygunClient.php]: I think `$this` is a good practice to follow for fluent setters today =)

File: src/Raygun4php/RaygunClient.php [hunk 2]
```diff
@@ -429,16 +439,16 @@ function filterParamsFromMessage($message, $replace = '[filtered]') {
...
      */
     function setFilterParams($params) {
       $this->filterParams = $params;
       return $this;
     }
 
     /**
-     * @return Array
+     * @return array
```
Comment (robbieaverill) [inline src/Raygun4php/RaygunClient.php]: Some of these changes have been merged elsewhere, will rebase for you

**Comment by robbieaverill** [comment]
Ok I can't rebase this pull request so I've resolved the conflicts and merged it by hand - see 81686f93321da6dc5ff2cce22db0b89fe03472d6.

Thanks for contributing!

**Pull request closed without merging**